settings.py

```diff
@@ -25,4 +25,8 @@
 
 INSTALLED_APPS = [
     "db",
```

Comment:
Created apps should be placed after system modules in `INSTALLED_APPS` as a general practice.